CHANGELOG.md

```diff
@@ -8,9 +8,13 @@
 
 - [Visualization previews are disabled.][1817] Previously, hovering over a
   node's output port for more than four seconds would temporarily reveal the
-  node's visualization. This behavior is disabled now.
+  node's visualization. This behavior is disabled now
```

Comment:
This needs to be moved to the "next relewase section".

Reply:
Fixed.